src/Symfony/Component/VarDumper/VarDumper.php

```diff
@@ -57,4 +65,12 @@ public static function setHandler(callable $callable = null)
 
         return $prevHandler;
     }
+
+    public static function setAfterDumpHandlerOnce(callable $callable = null)
```

Comment:
this looks like something we'd better avoid IMHO.
I think it's fine if we call `setEvaluationEnabled(true)` all the time when `dump()` is called.

Reply:
Sorry for getting back in touch so late. I'm aware this a not-important edge-case and I actually needed this at my old company and currently I don't. Anyways but I still want to finish this :)

Do you mean to adapt this callback to be called every-time instead of that only once logic which simplifies it or something completely different?